docs/spec/basic_facade_builder/add_facade.md

```diff
@@ -14,11 +14,11 @@ The alias template `add_facade` of `basic_facade_builder<Cs, Rs, MaxSize, MaxAli
 - sets `Copyability` to `std::max(Copyability, F::copyability)`, and
 - sets `Relocatability` to `std::max(Relocatability, F::relocatability)`, and
 - sets `Destructibility` to `std::max(Destructibility, F::destructibility)`, and
-- optionally, adds a convention for implicit upward conversion into `Cs` when `WithUpwardConversion` is `true`.
+- optionally, merges a direct convention of [`substitution_dispatch`](../substitution_dispatch/README.md) into `Cs` when `WithSubstitution` is `true`.
```

Comment:
nit: Maybe *merges **the** direct convention of ...*?

Reply:
Maybe not. If there's already a substitution convention for other facades, the new convention is expected to merge with the existing one.